src/Components/DropdownField.res

```diff
@@ -9,9 +9,17 @@ let make = (
   ~disabled=false,
   ~className="",
 ) => {
-  let {themeObj, localeString} = Recoil.useRecoilValueFromAtom(configAtom)
+  let {themeObj, localeString, config} = Recoil.useRecoilValueFromAtom(configAtom)
   let {readOnly} = Recoil.useRecoilValueFromAtom(optionAtom)
   let dropdownRef = React.useRef(Js.Nullable.null)
+  let (inputFocused, setInputFocused) = React.useState(_ => false)
+  let {parentURL} = Recoil.useRecoilValueFromAtom(keys)
+
+  let handleFocus = _ => {
+    setInputFocused(_ => true)
+    setValue(prev => prev)
```

Comment:
What is the use of this?

Reply:
i think i copied it from existing dropdown .. so no idea will have to figure out why this was being used